Documentation/Dev/Environment/Environment Setup.md

```diff
@@ -0,0 +1,43 @@
+# Environment Setup
+Steps required to setup and run the app in your local development environment.
+
+## First-Time Installations
+These steps are only required once for each computer you are working on.  
+Install Visual Studio Code: https://code.visualstudio.com/  
+Install Node.js: https://nodejs.org/en/
+
+## Updating your Local Version
+Every time you pull new changes into your local branch or do a first-time installation, you should do the following steps.
+
+### Client-side
+Open VS Code and open a new Terminal window. In the terminal, change to the `/client` directory and install the required client-side packages using `npm`:
+```cmd
```

Comment:
Looks good but maybe we can automate this a bit later.

Reply:
I looked into this and it looks like there's a long standing issue with batch files and NPM on Windows so I don't think it's worth the effort to try and find a work-around currently. 

See: https://github.com/npm/npm/issues/2938